src/keycloak/chart/templates/waypoint.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+{{- if .Capabilities.APIVersions.Has "gateway.networking.k8s.io/v1" }}
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: waypoint
```

Comment:
Definitely a nit but it might be worth naming this more specifically, `keycloak-waypoint` or something similar?

Reply:
Fixed. Thanks for spotting this.